shared/schemas/search-widget-data-params.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from 'zod';
+import {
+  VALID_SEARCH_WIDGET_DATA_OPERATORS,
+  WidgetDataFilterOperatorType,
+} from '@shared/dto/global/search-widget-data-params';
+
+export const SearchWidgetDataParamsSchema = z.object({
```

Comment:
```suggestion
export const WidgetDataFilterSchema = z.object({
```

or 

```suggestion
export const SectionFilterSchema = z.object({
```

Feels like a better naming. What do you think?

Reply:
I would go with WidgetDataFilterSchema

Reply:
Sounds ok to me, but then we are conceptually tying the Filters to the Widgets, and not to the sections, take that into account